src/processes/helper/createEmptyChannel.js

```diff
@@ -0,0 +1,28 @@
+const {CHANNEL_TYPE_STRING} = require('../../utils');
+const {LogError} = require('../../databases/models');
+const StreamChat = require('stream-chat').StreamChat;
+
+const createEmptyChannel = async (userId, usersFollowed) => {
+  try {
+    const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+
+    await Promise.all(
+      usersFollowed.map(async (userFollowed) => {
+        const members = [userId, userFollowed];
+        const channel = await client.channel(CHANNEL_TYPE_STRING.CHAT, {
+          members,
+          created_by_id: userId
+        });
+
+        await channel.create();
+      })
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createEmptyChannel` function is creating a new channel for each user followed. However, there's no error handling inside the `Promise.all`. If one of the promises fails, it will reject the entire operation and stop executing the remaining promises. Consider adding error handling inside the map function to ensure all channels are attempted to be created even if some fail.
```diff
-    await Promise.all(
-      usersFollowed.map(async (userFollowed) => {
-        const members = [userId, userFollowed];
-        const channel = await client.channel(CHANNEL_TYPE_STRING.CHAT, {
-          members,
-          created_by_id: userId
-        });
-
-        await channel.create();
-      })
-    );
+    await Promise.all(
+      usersFollowed.map(async (userFollowed) => {
+        try {
+          const members = [userId, userFollowed];
+          const channel = await client.channel(CHANNEL_TYPE_STRING.CHAT, {
+            members,
+            created_by_id: userId
+          });
+
+          await channel.create();
+        } catch (error) {
+          await LogError.create({
+            message: error.message
+          });
+          console.error(`Failed to create channel for user ${userFollowed}: `, error);
+        }
+      })
+    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->